Src/CZICmd/platform_defines.h

```diff
@@ -5,10 +5,9 @@
 #pragma once
 
 #if defined(_WIN32)
-// this means that the Win32-API is available, and the Windows-SDK is available
-#define WIN32ENV (1)
-#endif
-
-#if defined(__GNUC__)
-#define LINUXENV (1)
+    // this means that the Win32-API is available, and the Windows-SDK is available
+    #define WIN32ENV (1)
```

Comment:
Our usage of WIN32ENV checks whether it is defined #if defined(WIN32ENV) rather than true with  #if WIN32ENV. We may want to explicitly update our checks of the platform defines to check for truth rather than definition just as a matter of convention.

Reply:
yes, you are right, this all is fishy - but has to wait for some other PR :-)